### ISSUE TITLE
{"Response":"False","Error":"Error getting data."}

### ISSUE THREAD

**Issue by samir-nandardhane** [omdbapi/OMDb-API#286]
Hi  I am getting this below to so many IMDB IDs. below are the some IDs.
Issue :- {"Response":"False","Error":"Error getting data."}

IMDB_ID :- 

'tt12658966',
 'tt12658966',
 'tt12658964',
 'tt12658962',
 'tt12658960',
 'tt12658956',
 'tt12658954',
 'tt12658944',
 'tt12658942',
 'tt12658940',
 'tt12658938',
 'tt12658936',
 'tt12658934',
 'tt12658932',
 'tt12658930',
 'tt12658926',
 'tt12658924',
 'tt12658922',
 'tt12658920',
 'tt12658916',
 'tt12658912',
 'tt12658908',
 'tt12658906',
 'tt12658904',
 'tt12658902',
 'tt12658900',
 'tt12658886',
 'tt12658878',
 'tt12658876',
 'tt12658874',
 'tt12658872',
 'tt12658870',
 'tt12658864',
 'tt12658862',
 'tt12658860',
 'tt12658858',
 'tt12658856',
 'tt12658854',
 'tt12658850',
 'tt12658848',
 'tt12658842',
 'tt12658828',
 'tt12658814',
 'tt12658812',
 'tt12658810',
 'tt12658808',
 'tt12658806',
 'tt12658802',
 'tt12658800',
 'tt12658798',
 'tt12658790',
 'tt12658778',
 'tt12658774',
 'tt12658772',
 'tt12658770',
 'tt12658768',
 'tt12658764',
 'tt12658762',
 'tt12658760',
 'tt12658758',
 'tt12658756',
 'tt12658750',
 'tt12658746',
 'tt12658744',
 'tt12658742',
 'tt12658740',
 'tt12658736',
 'tt12658732',
 'tt12658730',
 'tt12658728',
 'tt12658726',
 'tt12658722',
 'tt12658720',
 'tt12658714',
 'tt12658712',
 'tt12658710',
 'tt12658708',
 'tt12658706',
 'tt12658702',
 'tt12658700',
 'tt12658696',
 'tt12658694',
 'tt12658688',
 'tt12658686',
 'tt12658682',
 'tt12658680',
 'tt12658678',
 'tt12658676',
 'tt12658670',
 'tt12658668',
 'tt12658662',
 'tt12658658',
 'tt12658656',
 'tt12658654',
 'tt12658650',
 'tt12658648',
 'tt12658646',
 'tt12658644',
 'tt12658642',
 'tt12658640',
 'tt12658638',
 'tt12658636',

**Comment by luisafalcao** [comment]
Same thing happening here. Did you figure it out?

**Comment by thebox-rj** [comment]
similar for tt26160190 ("Incorrect IMDb ID."), which is completely right.

**Comment by samir-nandardhane** [comment]
> Same thing happening here. Did you figure it out?

No, I tried some basic troubleshooting but no luck, did you find any thing

**Comment by nickflame20** [comment]
Same issue with tt27155038.

**Comment by sysadmjones** [comment]
getting an Incorrect IMDB ID for 'tt27489241' but it exists in IMDb.

**Comment by ds9001** [comment]
also for tt18305072

**Comment by AliDhiaus** [comment]
Response: 'False', Error: 'Incorrect IMDb ID. kalo kaya gini gmna saya cek di postman bisa codingannya pun gk ada yang salah tapi gini mulu
